specification/definitions/Webhook.json

```diff
@@ -15,14 +15,19 @@
     "attributes": {
       "type": "object",
       "required": [
+        "code",
         "name",
         "description"
       ],
       "additionalProperties": false,
       "properties": {
+        "code": {
+          "type": "string",
+          "example": "shipment_status_update"
```

Comment:
I created a `code` property here to identify the webhook, now we can keep the name and description more readable for humans and we can translate the name and description, but keep the `code` the same.

Reply:
This also had to change in the API. Is there a Utrecht item for this?